js/apps/admin-ui/public/locales/ja/authentication.json

```diff
@@ -31,9 +31,9 @@
   "webAuthnPolicyAcceptableAaguids": "許容可能なAAGUID",
   "default": "DEFAULT",
   "flow": {
-    "browserFlow": "ブラウザーフロー",
-    "registrationFlow": "登録フロー",
-    "directGrantFlow": "ダイレクト・グラント・フロー"
+    "browser": "ブラウザーフロー",
+    "registration": "登録フロー",
+    "direct grant": "ダイレクト・グラント・フロー"
```

Comment:
this is great, we should also add these but translated ` "reset credentials": "Reset credentials flow",
    "clients": "Client authentication flow",
    "docker auth": "Docker authentication flow"`

Reply:
@agagancarczyk - thank you for suggesting this, I would consider adding new keys out of scope for this: The scope of this PR is to update keys of already translated parts, so they are not lost. I would assume that the UI falls back on the original English values for those elements that don't have a translation.

Once we have Weblate in place, the translators will see a list of all untranslated strings, and can then provide a translation. I suppose this comment would be true for all other languages where you added a similar comment.

Reply:
@ahus1 Great to hear that Weblate will catch any untranslated strings as there will be a few. I'm going to approve your PR now. Thanks for fixing this!